packages/heartwood-components/components/06-modal/modal.scss

```diff
@@ -21,11 +21,14 @@
 
 .modal-header__title-wrapper {
 	display: inline-flex;
+	flex: 1;
 }
 
 .modal-header__title {
 	@include typescale('heading-lg');
 	margin-top: rem(2);
+	width: 100%;
+	text-align: center;
```

Comment:
@brandonminch I don't think this centering is gonna work, because the text is not visually centered when there is no back button. This would make a lot of our existing modals (especially longer titled ones) look lopsided. Also, I don't think this should be the default for _all_ modals — I'd rather configure it with something like `isCentered` on `Modal.Header` and `PagedModal` could just pass that in by default.

Reply:
whoops, yeah I meant to scope it to paged-modal but I'll make it a prop instead